src/assets/scss/mixin.scss

```diff
@@ -0,0 +1,30 @@
+@import './_variables.scss';
+@mixin d_flex {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+@mixin margin_auto{
+  margin: 0 auto;
+  position: relative;
+}
+@mixin position_rel{ position: relative;}
+@mixin position_abs_left {

```

Comment:
make class again this variable in the common.scss file so all the UI contributors will know that we have class for this functionality.